ixdiagnose/artifacts/logs.py

```diff
@@ -10,7 +10,9 @@ class Logs(Artifact):
         DirectoryPattern('ctdb'),
         DirectoryPattern('jobs'),
         DirectoryPattern('libvirt'),
-        DirectoryPattern('netdata'),
+        DirectoryPattern(
+            'netdata', pattern=r'^(access|error|debug|health)\.log$', max_size=2 * 1024 * 1024,
+        ),
         DirectoryPattern('openvpn'),
         DirectoryPattern('pods'),
```

Comment:
I think pods also showed up as being large. Did we check this directory too?

Reply:
@yocalebo i have not seen a case so far where pods directory is abusing specified size limits which can potentially cause problems..should we still go ahead and make relevant changes?